uwh-refbox/src/main.rs

```diff
@@ -140,53 +140,259 @@ fn main() -> std::result::Result<(), Box<dyn std::error::Error>> {
+                                .stroke(Some(timeout_color))
+                                .translate(Point::new(149, 4)),
+                        );
+                    }
+                    _ => display.draw(
+                        Font22x46::render_str("ERROR_3")
+                            .stroke(Some(red))
+                            .translate(Point::new(64, 18)),
+                    ),
+                };
+            }
+        };
 
-        display.draw(egline!((32, 32), (1, 32), stroke = Some(green)).translate(Point::new(64, 0)));
-        display.draw(egline!((32, 32), (40, 40), stroke = Some(blue)).translate(Point::new(64, 0)));
+        let black_penalty_count = 0;
+        let white_penalty_count = 0;
+
+        // EVERYTHING TO BE DISPLAYED ON THE BLACK SCORE PANEL
+        if black_penalty_count > 0 {
+            if state.b_score < 10 {
+                display.draw(
+                    Font32x64::render_str(&format!("{:>2}", state.b_score))
+                        .stroke(Some(blue))
+                        .translate(Point::new(2, 2)),
+                );
+            } else {
+                display.draw(
+                    Font22x46::render_str(&format!("{:<2}", state.b_score))
+                        .stroke(Some(blue))
+                        .translate(Point::new(11, 2)),
+                );
+            }
+        } else {
+            if state.b_score < 10 {
+                display.draw(
+                    Font32x64::render_str(&format!("{:<2}", state.b_score))
+                        .stroke(Some(blue))
+                        .translate(Point::new(2 + 16, 2)),
+                );
+            } else {
+                display.draw(
+                    Font32x64::render_str(&format!("{:<2}", state.b_score))
+                        .stroke(Some(blue))
+                        .translate(Point::new(2, 2)),
+                );
+            }
+        }
 
-        display.draw(
-            CustomFont6x8::render_str("12:34")
-                .stroke(Some(white))
-                .translate(Point::new(2, 8)),
-        );
-        display.draw(
-            Font8x15::render_str("12:34")
-                .stroke(Some(red))
-                .translate(Point::new(2, 16)),
-        );
-        display.draw(
-            Font11x25::render_str("12:34")
-                .stroke(Some(green))
-                .translate(Point::new(2, 31)),
-        );
-        display.draw(
-            Font16x31::render_str("12:34")
-                .stroke(Some(blue))
-                .translate(Point::new(64, 8)),
-        );
-        display.draw(
-            Font22x46::render_str("12:34")
-                .stroke(Some(white))
-                .translate(Point::new(64, 31)),
-        );
-        display.draw(
-            Font32x64::render_str("12:34")
-                .stroke(Some(red))
-                .translate(Point::new(128, 0)),
-        );
+        // EVERYTHING TO BE DISPLAYED ON THE WHITE SCORE PANEL
```

Comment:
Eventually this needs to handle b and w switching sides. Can you add an issue for this?

Reply:
I will think about that

Reply:
Can you add a github issue to track this?

Reply:
ok. done